src/main/play-doc/developer/CreatingBundles.md

```diff
@@ -154,9 +126,36 @@ The `javaOptions` values declare the maximum and minimum heap size for your appl
 
 `BundleKeys.memory` is used for locating machines with enough resources to run your application, and so it is particularly important to size it before you go to production.
 
+## Producing an OCI bundle with Docker
+ConductR is capable of running images that have been exported from Docker. To do this, you can use the `bndl` tool to convert a Docker image into a ConductR bundle. Behind the scenes, we're using the standard OCI Image format so you can be sure your images are free of vendor lock-in. Additionally, this approach doesn't require Docker to be installed in production; ConductR provides everything necessary to run these images.
```

Comment:
I would avoid "free of vendor lock-in". eg https://twitter.com/kelseyhightower/status/870313794525724672
I would suggest 'mitigate', 'control', 'manage' or otherwise that which improves your portability without potentially being inferred as disingenuous on the subject.